misc/python/materialize/scratch.py

```diff
@@ -398,6 +398,7 @@ def mssh(
     instance: Instance,
     command: str,
     *,
+    args: List[str] = [],
```

Comment:
Can we make the purpose of this a bit more clear?

```suggestion
    extra_ssh_args: List[str] = [],
```